RIPS/rip-77XX.md

```diff
@@ -49,8 +49,8 @@ The following factories should be added
 
 ### Why not align on one factory?
 
-The listed factories are already in active use on multiple networks. To ensure future compatibility without having to redeploy existing contracts it makes the most sense to enable a set of deployment factories that also cover a large part of the exsiting ecosystem.
+The listed factories are already in active use on multiple networks. To ensure future compatibility without having to redeploy existing contracts, it makes the most sense to enable a set of deployment factories that also cover a large part of the existing ecosystem.
 
 ## Backwards Compatibility
 
-No backward compatibility issues found as the precompiled contract will be added to `PRECOMPILED_ADDRESS` at the next available address in the precompiled address set.
+No backward compatibility issues were found.
```

Comment:
I'm not sure about this. What if a rogue actor steals one of the keys and deploys a different code on one of the networks? This is not backward compatibility per se, but I'm unsure how this would affect the proposal.